### ISSUE TITLE
Which loss function is used?

### ISSUE THREAD

**Issue by bssrdf** [fan-wenjie/LeNet-5#8]
Hi,

It appears the loss function is computed in the ```softmax``` function. Are you using MSE as the loss? Sorry, I couldn't figure it out from the code. 

Update: indeed ```softmax``` implicitly uses MSE as loss function. How to change it to cross-entropy loss?  

Thanks.
